rapidsai/build
# Add Quick Devel Build

As part of our efforts to ensure `devel` builds get published daily, we've added a `quick-devel` build type that builds a new image from the last known `devel` image. This will help debugging when `devel` images fail to build.

## **To Test**

To test this PR do the following:

- Check out the branch
- Run `./rapidsdevtool.sh genDockerfile -t ubuntu-quick` and ensure a Dockerfile is generated
- Run the following command to build the image

```
docker build --squash --build-arg CUDA_VERSION=10.0 --build-arg LINUX_VERSION=ubuntu18.04 --build-arg CXX_VERSION=7 --build-arg CC_VERSION=7 --build-arg PYTHON_VERSION=3.7 --build-arg PARALLEL_LEVEL=16 -f Dockerfile.ubuntu-quick .
```
